README.md

```diff
@@ -54,44 +72,52 @@ Here's an example webpack config illustrating how to use these options in your `
       // Required
       inject: false,
       template: require('html-webpack-template'),
-      //template: 'node_modules/html-webpack-template/index.ejs',
+      // template: 'node_modules/html-webpack-template/index.ejs',
 
       // Optional
+      appMountElement: 'article',
       appMountId: 'app',
       baseHref: 'http://example.com/awesome',
-      devServer: 3001,
+      devServer: 'http://localhost:3001',
```

Comment:
A port number by itself is interpreted as a relative path to a directory with that name.